bdtopo_v2/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/WorkflowBDTopo_V2.groovy

```diff
@@ -933,7 +933,7 @@ def extractProcessingParameters(def processing_parameters){
     def defaultParameters = [distance: 1000,indicatorUse: ["LCZ", "URBAN_TYPOLOGY", "TEB"],
                              svfSimplified:false, prefixName: "",
                              mapOfWeights : ["sky_view_factor"                : 4,
```

Comment:
I wonder whether in the OSM workflow (not here) the weights for "height of roughness" and "SVF" should be inverted (the first have 4 and the second have 6) because the SVF rely on building height but also building distribution while the height of roughness only rely on bulding height (which is roughly estimated...)